src/components/dialog/demoBasicUsage/script.js

```diff
@@ -57,30 +56,19 @@ angular.module('dialogDemo1', ['ngMaterial'])
   };
 
   $scope.showAdvanced = function(ev) {
-    var useFullScreen = ($mdMedia('sm') || $mdMedia('xs'))  && $scope.customFullscreen;
-
     $mdDialog.show({
       controller: DialogController,
       templateUrl: 'dialog1.tmpl.html',
       parent: angular.element(document.body),
       targetEvent: ev,
-      clickOutsideToClose:true,
-      fullscreen: useFullScreen
+      clickOutsideToClose: true,
+      fullscreen: true // Fullscreen only applies for -xs, -sm breakpoints.
```

Comment:
Edit: Just rethought again. We don't want to force the users on -xs, -sm breakpoints to always use the Fullscreen Dialog.

Those should be able to toggle that functionality within the `checkbox` in the demo HTML.